src/main/java/com/rickmorty/Controllers/UserController.java

```diff
@@ -0,0 +1,76 @@
+package com.rickmorty.Controllers;
+
+import com.rickmorty.Models.UserModel;
+import com.rickmorty.Services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    // Cria um novo usuário
+    @PostMapping
+    public ResponseEntity<?> createUser(@RequestBody UserModel user) {
+        if (userService.existsByEmail(user.getEmail())) {
+            return ResponseEntity
+                    .status(HttpStatus.BAD_REQUEST)
+                    .body(Map.of("error", "Email já cadastrado.", "code", 400));
+        }
+
+        LocalDate currentDate = LocalDate.now();
+        user.setDate_register(currentDate);
+        user.setDate_update(currentDate);
+
+        UserModel createdUser = userService.saveUser(user);
+        return new ResponseEntity<>(createdUser, HttpStatus.CREATED);
+    }
+
+    // Obtém todos os usuários
```

Comment:
Remove os comentários e tudo o que for pertinente essa rota para pegar todos os usuários, tanto no controller quanto no service e repositório, tua atividade tá no escopo de criação de usuário, ver usuários, atualização parcial ou total ou delete tão fora do escopo desse card.

Reply:
Foi excluído doto CRUD e mantido apenas o POST.